services/ui-src/src/types/reports.ts

```diff
@@ -220,6 +254,14 @@ export interface OverlayReportPageVerbiage extends ReportPageVerbiage {
   enterEntityDetailsButtonText: string;
 }
 
+export interface PlanOverlayReportPageVerbiage extends ReportPageVerbiage {
+  requiredMessages: {
+    [key: string]: CustomHtmlElement[] | undefined;
+  };
+  tableHeader: string;
+  enterEntityDetailsButtonText: string;
+}
```

Comment:
extends `ReportPageVerbiage` which includes the `intro: {}` object types
![Screenshot 2025-04-15 at 9 59 45 AM](https://github.com/user-attachments/assets/e5ba8767-ecc0-47b8-b122-04921973864c)